main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/faraonc/hwsc-user-svc/logtag"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	pb "github.com/faraonc/hwsc-api-blocks/int/hwsc-user-svc/proto"
+	svc "github.com/faraonc/hwsc-user-svc/service"
+)
+
+const (
+	connectType = "tcp"
```

Comment:
This is on me, but a configuration management will be needed in the future. We might as well start doing this practice. Please review this PR. https://github.com/faraonc/hwsc-document-svc/pull/5